pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/showcases/misc/UpdatableMediaItemShowcase.kt

```diff
@@ -32,4 +33,10 @@ fun UpdatableMediaItemShowcase() {
             )
         }
     }
+    LifecycleResumeEffect(player) {
```

Comment:
Shouldn't we also `release` the player?

Reply:
It is already released inside the associated ViewModel `UpdatableMediaItemViewModel`